06-building-a-program-start-to-finish/writing-tests/output/spec/invertedSpec.js

```diff
@@ -0,0 +1,257 @@
+const newIndex = new InvertedIndex();
```

Comment:
`invertedSpec.js` is in violation of proper naming conventions for test files. Please ask your peers for what is preferred.